script/1_0_2current_en_script/Ciel/Day 5/05_01_CIEL05_1DASH.txt

```diff
@@ -34,75 +34,75 @@ The number of people receiving these messages is unspecified, but they are conti
 <Page11>
 As it turns out, there's nothing unusual about the department store.
 From the grocers in the basement to the speciality shops on the seventh floor, it's an ordinary department store.
-But...
+Except...
 <Page12>
 "...There isn't anything above the seventh floor. Considering the size of this department store, it should have at least ten floors..."
 <Page13>
-The elevator only has buttons to go up to the seventh floor, and the stairs have a sign saying that the seventh floor is off-limits.
+The elevator only has buttons to go up to the seventh floor, and once you reach the seventh floor landing on the stairs there's a sign saying that anything further is off-limits.//階段は七階に立ち入り禁止の立て札が置かれていた。
 <Page14>
-Aiming for a moment when no one is around, I climb the stairs, slipping past the no-entry rope
+Aiming for a moment when no one is around, I slip past the no-entry rope and climb up the stairs.
 The stairs lead up to the upper floors, but I'm blocked.
-The part leading to the eighth floor has shutters down as expected.
+The path leading to the eighth floor is shuttered, as expected.//八階のフロアに通じる部分には、当然のようにシャッターが下ろされている。
 Next to it, there is a locked emergency door.
 <Page15>
-"...If there's anything here, it's this way..."
+"...If there's anything here, it's on the other side of this door..."//Reified for flow 何かあるとしたら、この向こうか
 I ponder whether to go inside or to return back.
 <Page16>
-I know I'm going to do something reckless.
+I know that what I'm trying to do is reckless.//自分が無謀な事をしようとしているのは理解している。
 I'll be entering that monster's hideout alone.
-I know very well,
+I've experienced firsthand, time and again,//それがどれほどいかれた事かは、俺はもう、何度も味わって知っている。
 how crazy such a move is.//I've inverted 53 and 54
 <Page17>
 "...It's okay. I should be able to do it."
 I clutch the knife hidden in my pocket.
 I've got these eyes if it comes down to it.
 <Page18>
 I thought about running to the police and explaining the situation. It's the safest thing to do.
-But what would that achieve? In the first place, I'm going to kill him. How is a killer asking the police for help going to persuade anyone? To begin with, we probably don't share the same goal.
+But what would that achieve? In the first place, I'm going to kill him. How is a killer supposed to be able to persuade the police to help him? There's no way that those two groups could share a common purpose.//けどそれでどうなる？　そもそも俺はこれから“殺し”に行くんだ。そんな殺人者が警察に駆けこんだところで誰を説得できるのか。そもそも目的を共有できるはずがない。
```

Comment:
What are "those two groups" here?

Reply:
the police + murderers... not the best phrasing, will sleep on a better way to word

Reply:
Does this read OK? It does conjure a bit of context.
> There's no way such natural adversaries could share a common purpose.